index.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const bodyParser = require('body-parser')
+const mongoose = require("mongoose")
+const cors = require("cors")
+
+const app = express();
+
+app.use(bodyParser.json());
```

Comment:
body parser is now available in express , trying that would result in small yet counted optimization

instead of
```
app.use(bodyParser.json())
```

try that
```
app.use(express.json())
```